src/pages/inventory/inventory.ts

```diff
@@ -3,18 +3,22 @@ import { NavController } from 'ionic-angular';
 
 import { InventoryData } from '../../providers/inventory-data';
 import { StockpileData } from '../../providers/stockpile-data';
-import { Statuses, Actions } from '../../constants';
+import { Actions } from '../../constants';
 import { ItemPage } from '../item/item';
 
 @Component({
   selector: 'page-inventory',
   templateUrl: 'inventory.html'
 })
 export class InventoryPage {
-  statuses = Statuses;
-  segment = Statuses.all;
+  segment = -1;
```

Comment:
It works in components with the `===` comparison, but it does not bind values in the template correctly so the default options in the segment and dropdowns are never highlighted. Probably because they don't consider `null` as a value.